server/fishtest/templates/user.mak

```diff
@@ -20,7 +20,7 @@
   <div class="control-group">
     <label class="control-label">New password:</label>
     <div class="controls">
-      <input name="password" type="password"/>
+      <input name="password" type="password" pattern=".{8,}" title="Eight or more characters"/>
```

Comment:
"Eight or more non trivial characters: password too simple will be rejected"

Reply:
I agree. Even though we display a warning notification which explains why the password is rejected, it's better to give a heads up to the user. I think the signup page should also incorporate the same type of warning here:
https://github.com/glinscott/fishtest/blob/master/server/fishtest/templates/signup.mak#L28

Reply:
I agree with the signup page improvement.

Reply:
Pushed changes for both pages, thank you